tox.ini

```diff
@@ -20,3 +20,7 @@ commands = flake8 pyquil
 [flake8]
 ignore = E501,E999,E741,E126,E127,F401,F403,F405,F811,F841,E743,W503
 exclude = gen3,external
+
+[testenv:mypy]
+# The dream is to one day run mypy on the whole tree. For now, limit checks to known-good files.
+commands = mypy pyquil/quilatom.py pyquil/quilbase.py pyquil/gates.py
```

Comment:
im going to delete the `tox.ini` soon just as an FYI, but im fine leaving the change here